src/server/api/routers/auth.ts

```diff
@@ -127,13 +136,21 @@ export const authRouter = createTRPCRouter({
           success: true,
         };
       } catch (error) {
+        if (error instanceof z.ZodError) {
+          throw new TRPCError({
+            code: 'BAD_REQUEST',
+            message: 'Input validation error',
+            cause: error,
+          });
+        }
+
```

Comment:
I think we should keep the default error for validation. I don't think it'll be thrown in this function anyway, so I don't believe this if statement will ever be hit anyway.

Reply:
Think it should be good!

Reply:
Hey Joy, I meant that we should remove these lines since it's dead code (will never be hit), and we also wouldn't even want to reformat the error this way. Sorry if this wasn't clear

Reply:
Wait I thought I removed it in my last commit

Reply:
Fixed mbb, idk what's up with prettier :(